app/lib/efile/ny/it201.rb

```diff
@@ -326,7 +326,7 @@ def calculate_line_69a
       end
 
       def calculate_line_72
-        total_state_taxes_withheld = @direct_file_data.total_state_tax_withheld
+        total_state_taxes_withheld = @direct_file_data.total_w2_state_tax_withheld
         state_file_1099gs = @intake.state_file1099_gs
```

Comment:
Should this get refactored the same way `Az140Calculator#calculate_line_53` did (to use `@intake.state_file1099_gs.sum(&:state_income_tax_withheld)`? And should we add `@direct_file_data.total_1099r_state_tax_withheld` here too, as was done there? Not sure what our strategy is re: working on NY code at this point.

Reply:
I am happy to refactor NY, but I don't feel as comfortable adding 1099R in just this one place without adding it elsewhere too